docs/assets/js/swagger.yml

```diff
@@ -236,6 +242,10 @@ components:
         size_bytes:
           type: integer
           format: int64
+        mtime:
```

Comment:
you're going to publish an API that hasn't been released yet (well, same for yours)...

Reply:
Reverted.